src/schema/contentful/phoenix.js

```diff
@@ -60,7 +70,7 @@ const typeDefs = gql`
     url(w: Int, h: Int, fit: ResizeOption): AbsoluteUrl,
   }
 
-  type CampaignWebsite {
+  type CampaignWebsite implements Showcasable{
```

Comment:
Small visual gripe - but let's add a space between the last word and the curly bracket
```suggestion
  type CampaignWebsite implements Showcasable {
```

Let's do the same for the other ones as well.